.env

```diff
@@ -12,15 +12,15 @@ INTELOWL_TAG_VERSION=v1.9.0
 ###### Default (Production) ######
 COMPOSE_FILE=docker-compose.yml
 # To run all additional integrations in production
-#COMPOSE_FILE=docker-compose.yml:./integrations/docker-compose.peframe.yml:./integrations/docker-compose.thug.yml:./integrations/docker-compose.capa.yml:./integrations/docker-compose.boxjs.yml
+#COMPOSE_FILE=docker-compose.yml:./integrations/docker-compose.thug.yml:./integrations/docker-compose-for-tests.exe_analyzers.yml:./integrations/docker-compose.boxjs.yml
```

Comment:
In this line, it should be `docker-compose.exe_analyzers.yml`, without `for-tests` part

Reply:
Meanwhile, I also noticed that we are missing the `docker-compose.apk.yml` entry. on line 20 and 23 for the test compose files as well. We should add this in another commit.

Reply:
The last thing was because I found unresolved problems with the logs on a clean startup. So I disabled that container for the moment. I did not open any specific issue, while I should have did it

Reply:
@mlodic, Could you explain more about your problem with logs? I too encountered one at the start. 

It's log directory wasn't creating in `/var/log/intel_owl`. I don't know why this happened, I created in manually by entering into the docker shell. Is this you faced?

But from the next clean build (Deleted local volumes of previous), it worked fine.

Still, I didn't understand why it behaved like that. 

Is this what you faced?

Reply:
Yes, this problem happens sometimes. It's something to do with permissions which gets changed on changing the log directories and files.

Reply:
So, Shall I add both changes into one? It's a small change and not needed of one extra commit, IMO

Reply:
It's a small change but we should still have it in a different commit/PR since it's not related to changes introduced in this particular pull request and then it's also easier to track.